README.md

```diff
@@ -52,29 +53,29 @@ challenges raised by the [CAP Theorem][]. Each application
 can select the right combination of libraries to solve its particular
 distributed problem.
 
-Irmin consists of a core of well-defined low-level data structures
-that specify how data should be persisted and be shared across
-nodes. It defines algorithms for efficient synchronization of those
+Irmin is built on a core of well-defined, low-level data structures that 
+dictate how data should be persisted and shared across nodes. 
+It defines algorithms for efficient synchronisation of those
 distributed low-level constructs. It also builds a collection of
 higher-level data structures that developers can use without knowing
 precisely how Irmin works underneath. Some of these components even
-have a [formal semantics][], including [Conflict-free Replicated
+have [formal semantics][], including [Conflict-Free Replicated
```

Comment:
```suggestion
have [formal semantics][], including [Conflict-free Replicated
```
(the weird capitalization is there because the acronym is CRDT and not CFRDT)